lib/crawler_web/controllers/keyword_controller.ex

```diff
@@ -0,0 +1,73 @@
+defmodule CrawlerWeb.KeywordController do
+  use CrawlerWeb, :controller
+
+  alias Crawler.Keyword.Helpers.CSVParser
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.KeywordCSVFile
+
+  def create(conn, %{"keyword_csv_file" => params}) do
+    changeset = %{
+      KeywordCSVFile.create_changeset(%KeywordCSVFile{}, params)
+      | action: :validate
+    }
+
+    case changeset.valid? do
+      true ->
+        process_uploaded_file(conn, changeset.changes)
+
+      false ->
+        show_error_flash_message_and_redirects_to_dasboard(
+          conn,
+          gettext("The file doesn't look like CSV")
+        )
+    end
+  end
+
+  defp process_uploaded_file(conn, changes) do
+    case CSVParser.parse(changes.file.path) do
+      {:ok, keyword_list} ->
+        {num_keyword, _list} =
+          Keywords.create_keyword_list(keyword_list, conn.assigns.current_user.id)
+
+        conn
+        |> put_flash(:info, "#{num_keyword} #{gettext("keywords were uploaded!")}")
```

Comment:
We  can move the `num_keyword` into the gettext` 

```suggestion
        |> put_flash(:info,  gettext("%{num_keyword} keywords were uploaded!", num_keyword: num_keyword))
```

Reply:
Updated in [d493a8c](https://github.com/vnntsu/elixir-ic/pull/36/commits/d493a8cb0958563e30008880bc6ebf7531acadb7)